specification/resources/resource-manager/Microsoft.Authorization/stable/2021-06-01/dataPolicyManifests.json

```diff
@@ -0,0 +1,311 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "PolicyClient",
+    "version": "2021-06-01",
```

Comment:
did you mean to pull this forward?  The manifest doesn't have this version for dataPolicyManifests right?  I'm not against it but check with dataplane and add to manifest if we do this

Reply:
I did initially, but chatted with Varun and he didn't want me to do it.  So I reverted it.